.github/workflows/test.yml

```diff
@@ -333,7 +333,7 @@ jobs:
       - uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5.0.2
         with:
           go-version: ${{ env.GO_VERSION }}
-      - name: Run Govulncheck
-        run: make verify-govulncheck
-      - name: Run Gosec
-        run: make verify-gosec
+      - name: Run govulncheck
+        run: sudo -E make verify-govulncheck
+      - name: Run gosec
+        run: sudo -E make verify-gosec
```

Comment:
Let's get https://github.com/cri-o/cri-o/pull/8573 in before we can verify if that works.

Reply:
Sure!

Reply:
Now that https://github.com/cri-o/cri-o/pull/8573 has been merged. I rebased against `main`.